src/sass/Fabric.Color.Mixins.Output.scss

```diff
@@ -15,98 +15,98 @@
   @include ms-bgColor-themeDark;
 }
 
-.ms-bgColor-themeDarkAlt, 
```

Comment:
Thanks for cleaning this file up! I'll approve these changes for this pull request, but in the future, it'd be good to keep changes scoped to just what's in the current review. Preferably, there would be a second PR for the changes in this file, so that these changes can be tracked separately of the MessageBar work.